jetty-bom/pom.xml

```diff
@@ -210,6 +210,7 @@
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>infinispan-remote</artifactId>
+        <classifier>config</classifier>
```

Comment:
What?

Reply:
[https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/

[https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/infinispan-remote-10.0.11-<b>config</b>.jar](https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/infinispan-remote-10.0.11-config.jar)

Reply:
Depending on the `jetty-home` `config` classifier in a bom is just super odd, and feels wrong.
Why not just have `<type>pom</type>` instead?

Reply:
Yes, you are right, `<type>pom</type>` is better.

Reply:
@olamy can you think of any maven technical reason for not just using `<type>pom</type>` instead?

Reply:
the bom is mostly made (e.g only useful) for embedded users.
So TBH I don't even understand why we have `<artifactId>infinispan-remote</artifactId>` in the bom. This module contains no code but only configuration files.
So except the `config` jar which contains the jetty configuration module files it's a totally useless artifact :) 
Either we remove it from the bom or we "expose" the only useless artifact which is the `config` but frankly I even don't understand why :) 
wdy? Remove the entry from the bom?

Reply:
> Remove the entry from the bom?

I'd vote +1 for this approach.

Reply:
FTR https://github.com/eclipse/jetty.project/pull/8554